lib/plugins/aws/deploy/compile/functions/index.js

```diff
@@ -183,6 +176,42 @@ class AwsCompileFunctions {
 
       _.merge(this.serverless.service.provider.compiledCloudFormationTemplate.Outputs,
         newOutputObject);
+
+      if (typeof this.serverless.service.provider.iamRoleARN !== 'string') { // only if default IAM
+        const logRetentionInDays = 14; // TODO make configurable?
+        const logGroupTemplate = `
+          {
+            "${normalizedFunctionName}LogGroup": {
+              "Type" : "AWS::Logs::LogGroup",
+              "Properties" : {
+                "LogGroupName" : "/aws/lambda/${FunctionName}",
+                "RetentionInDays" : "${logRetentionInDays}"
```

Comment:
I don't think we should set a log retention policy unless its configurable. Otherwise that might be a surprise for people.